backend/layers/processing/publish_revisions.py

```diff
@@ -92,8 +92,8 @@ def run(self):
                     )
                     try:
                         self.business_logic.publish_collection_version(collection_version.version_id)
-                    except Exception:
-                        logging.exception("Failed to publish collection version.")
+                    except Exception as e:
```

Comment:
This is redudant. [Logging.exception](https://docs.python.org/3/library/logging.html#logging.Logger.exception) includes the exception.